src/pages/login/register.vue

```diff
@@ -33,6 +33,25 @@
     <div class="error">
       {{ getError('password2') }}
     </div>
+    <Checkbox v-model="form.acceptCondition">
+      {{ $t('accept') }}
+      <a
+        href="https://ocial.es/privacy"
+        style="color:var(--o-color-theme)"
+        target="_blank">
+        {{ $t('policy') }}
+      </a>
+      {{ $t('and') }}
+      <a
+        href="https://ocial.es/customeragreement"
+        style="color:var(--o-color-theme)"
+        target="_blank">
+        {{ $t('terms') }}
+      </a>
+    </Checkbox>
```

Comment:
Usad interpolaciones para las traducciones: https://vue-i18n.intlify.dev/guide/essentials/syntax.html#interpolations

Reply:
viendo la guía, no me queda claro como meter interpolaciones con las <a> de por medio. ¿Cómo se haría?

Reply:
@Jeszamgue Perdón, el enlace más correcto era este: https://vue-i18n.intlify.dev/guide/advanced/component.html#component-interpolation

En Basic Usage tenéis un ejemplo directo.

Reply:
he seguido la guía y no sale ningún mensaje. También probé a copiar y pegar el código de ejemplo y no muestra ningún mensaje tampoco. Así que lo dejaré así, que no puedo estar más tiempo con esto.